server/routes/upload.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const router = express.Router();
+const multer = require('multer');
+const multerS3 = require('multer-s3');
+const aws = require('aws-sdk');
+
+const s3 = new aws.S3({
+	accessKeyId: process.env.AWS_KEY,
+	secretAccessKey: process.env.AWS_SECRET
+});
+
+const upload = multer({
+	storage: multerS3({
+		s3,
+		bucket: process.env.BUCKET,
+		acl: 'public-read',
```

Comment:
I did some research on the ACL but I am not sure what to set the permissions for this, and the permissions for the bucket